test/e2e/clients_test.go

```
@@ -0,0 +1,28 @@
+var (
+	publicMobileClient = resty.New().
+				SetBaseURL(PublicURL + "/mobile/v1").
+				SetTimeout(300 * time.Millisecond)
+	privateMobileClient = resty.New().
+				SetBaseURL(PrivateURL + "/mobile/v1").
+				SetTimeout(300 * time.Millisecond)
+
+	publicUserClient = resty.New().
+				SetBaseURL(PublicURL + "/3rdparty/v1").
+				SetTimeout(300 * time.Millisecond)
+	privateUserClient = resty.New().
```

golangci-lint (1.64.8)

var `privateUserClient` is unused

(unused)